### PR TITLE
Update draft -- Sriram's edits

### PR DESCRIPTION
**Pull request by ksriram25**
Went through Jeff's changes carefully and did some of my own edits.
Added new wording to clarify the text in some places.
Fixed an error in example B.2. Tweaked it a bit.

Alvaro's comment at the mike (IETF 115)? What wording if any do we want to add?

### CODE REVIEW

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Please preserve indentation.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Wrap line.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: The change above seems right.  Remove comment.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Fix indentation.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Not sure why the line split here.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Fix indentation.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Fix indentation and line wrap.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml:523]: The error here you're noting is valid.
The intent is to have an example with a common leading AS with trailing AS that is thrown into a set that is discarded.  You can either update this in your diff, or leave it for me to update in mine once we merge your stuff in.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml:545]: brief-style?

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: indentation.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: I would skip this parenthetical (and mis-indented) statement.  It doesn't necessarily help with the explanation, and the "beginning" portion isn't how we'd normally refer to this in RFC land.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (jhaas-pfrc) [inline draft-ietf-idr-deprecate-as-set-confed-set.xml]: Fix indentation, keep this.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: OK, sure.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml:523]: Yes, your origin example B.2 didn't follow this condition from rfc 4271:

           - all tuples of type AS_SEQUENCE in the aggregated AS_PATH
             SHALL appear in all of the AS_PATHs in the initial set of
             routes to be aggregated. 

As I see it B.3 is already "an example with a common leading AS with trailing AS that is thrown into a set that is discarded."
I leave this up to you to sort out.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml:545]: You were varyingly using "brief aggregation", "brief-style aggregation" and "brief style aggregation" but all meant the same. I tried to retain only one form "brief aggregation". Eliminated the attachment of 'style'.

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: fixed

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: fixed

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

File: draft-ietf-idr-deprecate-as-set-confed-set.xml
Comment (ksriram25) [reply draft-ietf-idr-deprecate-as-set-confed-set.xml]: done

**Review by jhaas-pfrc — APPROVED** [review]
Again, approve, merge and sort out the lingering issues in -10 branch.

**Pull request merged**
